spec/project/project_spec.rb

```diff
@@ -116,19 +115,19 @@
 
       # Add additional user while changing Buffalos surname and role.
       bill.last_name = 'Billie'
-      other_guy = ProjectHelper.create_random_user(@client)
-      additional_batch = [bill, other_guy]
+      additional_batch = [bill, other_user]
 
       @domain.create_users(additional_batch, domain: @domain)
       @project.import_users(additional_batch.map { |u| { user: u, role: u.role } }, domain: @domain, whitelists: [/admin@gooddata.com/])
 
-      expect(@project.members.count).to eq 4
+      expect(@project.members.count).to eq additional_batch.count + 1
       expect(@project.member?(bill)).to be_truthy
       expect(@project.members?(users - additional_batch).any?).to be_falsey
     end
 
     it "Updates user's role in a project" do
-      users = (1..5).to_a.map { ProjectHelper.create_random_user(@client).to_hash }
+      users = ProjectHelper.ensure_users(client: @client, amount: 5, caller: $example_name).map &:to_hash
```

Comment:
Lint/AmbiguousOperator: Ambiguous block operator. Parenthesize the method arguments if it's surely a block operator, or add a whitespace to the right of the `&` if it should be a binary AND.